native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/nats-native/reflect-config.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name":"io.nats.client.impl.SocketDataPort",
```

Comment:
Can we format this json like this? :
```
[
    {
        "name": "io.nats.client.impl.SocketDataPort",
        "methods": [
            {
                "name": "<init>",
                "parameterTypes": []
            }
        ]
    }
]
```

Reply:
Fixed in f4d3d36f778fcfa3e85ee7bdcbbfb4ad853f9c00